src/components/UI/AttachmentLink/AttachmentLink.tsx

```diff
@@ -1,11 +1,13 @@
-import styled from "@emotion/styled";
 import { getData, v2, WrappedDocument } from "@govtechsg/open-attestation";
 import prettyBytes from "pretty-bytes";
 import React, { FunctionComponent } from "react";
 import { Paperclip } from "react-feather";
-import tw from "twin.macro";
 import { getLogger } from "../../../utils/logger";
 
+export enum NestedDocumentState {
+  LOAD = "NESTED_DOCUMENT_LOAD",
+}
+
```

Comment:
better to put this in a common area then exporting it from here.

any other states besides `LOAD`?

Reply:
no other states yet atm from wat i see, ok will shift